crates/zk-por-core/src/config.rs

```diff
@@ -32,6 +32,7 @@ pub struct ConfigProver {
     pub batch_size: usize,
     pub num_of_tokens: usize,
     pub user_data_path: String,
+    pub num_users: usize,
```

Comment:
this seems not to be used anywhere; can remove that

Reply:
Yeah I was refactoring the cfg regarding the metadata but didnt want to effect downstream changes, forgot to remove.